JobConfig/beam/DS-flash.fcl

```diff
@@ -59,7 +57,7 @@ physics.producers.g4run.SDConfig.preSimulatedHits:  ["flashResample:virtualdetec
 physics.producers.stoppedMuonFinder: {
    module_type: "StoppedParticlesFinder"
    particleInput: "g4run"
-   particleOffsetThreshold: 300000
+   simStageThreshold: 3
```

Comment:
What is the accounting that gives the value 3 here?  Don't you have to assume something about prior processing?

Reply:
That was a bug.  I have just pushed a fix. The intent is to veto daughters of muons stopped in the current stage  (they are statistically accounted for by mu stop resampling, so we want to exclude them from the flash resampling).   With the current set of job configs we have: PS simStage=0, TS simStage=1, DS and DS-flash simStage=2.   If we eliminate a stage this threshold will need to be adjusted.

Reply:
The stage threshold setting seems a fragile part of this design: it depends on production details, and (AKAIK) will fail silently if set wrong.  I think a stronger design is to explicitly select a separate collection of StepPoints that are exactly the ones that need to be continued, and not rely on any automatic selection.  As long as the current code supports that (ie that I can set the threshold to 'accept everything') I think it is OK to have this feature.

Reply:
Dave, thank you for another useful comment.  I realized that we have enough information to do the right thing without setting the stage threshold by hand.   A commit removing it from JobConfig has been just pushed.